ref/license-information.md

```diff
@@ -0,0 +1,8 @@
+Anbox Cloud is built using multiple open source software components. This guide lists those components and where to find their license information.
+
+| Software components | Where to find their licenses |
+|--|--|
+| LXD images | Within the LXD image, <br/>`/usr/share/licenses` and `/opt/mesa/share/licenses`<br/> The copyright information of Anbox Cloud is available at `/usr/share/doc/*/copyright`. |
```

Comment:
We're not there yet for mesa licenses until https://warthogs.atlassian.net/browse/AC-1661 get landed. If we're going to publlish this after it's merged, we'd better drop it